modules/ocf/manifests/base.pp

```diff
@@ -0,0 +1,19 @@
+class ocf::base {
```

Comment:
can we just go back to having this be `class ocf` (in the `init.pp` file)? that's the pattern we use elsewhere and what we used to do here. in retrospect i think it was a mistake for us to move this list into hiera, glad we're moving it back

Reply:
Yeah, that's fair, probably a better naming scheme than this.